src/api/app/helpers/webui/package_helper.rb

```diff
@@ -87,7 +87,8 @@ def repo_type_and_priority(repository)
     [repository.repo_type, repository.priority].compact.join(', Priority: ')
   end
 
-  def cloud_image_file?(filename)
-    filename.end_with?('.raw.xz', '.vhdfixed.xz')
+  def uploadable?(filename, architecture)
+    ::Cloud::UploadJob.uploadable_file?(filename) &&
+      ::Cloud::UploadJob.uploadable_architecture?(architecture)
```

Comment:
[Nitpick/Suggestion] If you were to use bjoern's approach (individual param attributes on the UploadJob class) then you could do something like this here:
```ruby
def uploadable?(filename, architecture)
  ::Cloud::UploadJob.new(filename: filename, architecture: architecture).uploadable?
end
```
So you'd get rid of the duplicated logic.